staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1523,6 +1523,8 @@ message ContainerStats {
     MemoryUsage memory = 3;
     // Usage of the writable layer.
     FilesystemUsage writable_layer = 4;
+    // Usage of the Block IO
+    BlockIOUsage block_io_usage = 5;
```

Comment:
```suggestion
    BlockIOUsage block_io = 5;
```
noting above cpu and memory field name pattern..

Reply:
same for v1alpha proto

Reply:
Thank you for your suggestion. The modification has been committed.

Reply:
I am still seeing `block_io_usage` here...